runtime/pangolin-parachain/src/pallets/remote_government.rs

```diff
@@ -0,0 +1,46 @@
+// --- paritytech ---
+use frame_support::traits::{EnsureOrigin, Get};
+use frame_system::RawOrigin;
+// --- darwinia-network ---
+use crate::*;
+use dp_common_runtime::remote_government::Config;
+
+pub struct EnsureSpecific;
+impl<O: Into<Result<RawOrigin<AccountId>, O>> + From<RawOrigin<AccountId>>> EnsureOrigin<O>
+	for EnsureSpecific
+{
+	type Success = AccountId;
+
+	fn try_origin(o: O) -> Result<Self::Success, O> {
+		o.into().and_then(|o| match o {
+			RawOrigin::Signed(who)
+				if who
+					== array_bytes::hex_into_unchecked(
+						"0x72819fbc1b93196fa230243947c1726cbea7e33044c7eb6f736ff345561f9e4c",
+					) =>
+				Ok(who),
+			r => Err(O::from(r)),
+		})
+	}
+}
+
+pub struct PangolinBestFinalized;
+impl Get<Hash> for PangolinBestFinalized {
+	fn get() -> Hash {
+		<pallet_bridge_grandpa::BestFinalized<Runtime, WithPangolinGrandpa>>::get()
+	}
+}
+
+frame_support::parameter_types! {
+	pub const CheckInterval: BlockNumber = 2 * HOURS;
```

Comment:
```suggestion
	pub const CheckInterval: BlockNumber = 6 * HOURS;
```

Reply:
Each test takes 6 hours.
Should we change this?

Reply:
Ah, this is for pangolin, not Crab.

No need to change then